src/main/java/io/github/syst3ms/skriptparser/parsing/SyntaxParser.java

```diff
@@ -107,7 +110,10 @@
             if (variable.filter(v -> !v.isSingle() && expectedType.isSingle()).isPresent()) {
                 logger.error("A single value was expected, but " +
                         s +
-                        " represents multiple values.", ErrorType.SEMANTIC_ERROR);
+                        " represents multiple values.",
```

Comment:
This is only one example, and I admit that I wasn't consistent with it even before this commit, but there should be a line break after the method's opening parenthesis.